cmd/routed-eni-cni-plugin/driver/driver.go

```diff
@@ -387,8 +394,15 @@ func (n *linuxNetwork) setupVeth(hostVethName string, contVethName string, netns
 		}
 		log.Debugf("Successfully deleted old hostVeth %s", hostVethName)
 	}
-
-	createVethContext := newCreateVethPairContext(contVethName, hostVethName, ipAddr, mtu, index)
+	macAddrStr, err := NewMACGenerator().generateUniqueRandomMAC()
```

Comment:
nit: I'd like to let generateUniqueRandomMAC return a net.HardwareAddress directly, so it frees the consumer code to parse and handle error.